docker/Dockerfile

```diff
@@ -63,7 +63,7 @@ CMD ["/usr/local/bin/nginx.sh"]
 # zipkin-slim - An image containing the slim distribution of Zipkin server.
 #####
 
-FROM openzipkin/jre-full:11.0.6-11.37 as zipkin-slim
+FROM openzipkin/jre-full:11.0.7-11.39.15 as zipkin-slim
```

Comment:
odd but correct due to our naming conventions :) 

https://github.com/zulu-openjdk/zulu-openjdk/tree/master/11.0.7-11.39.15
https://github.com/openzipkin/docker-jre-full/tree/11.0.7-11.39.15

Reply:
odd that they appended another .NN I mean